plugins/lookup/github_app_access_token.py

```diff
@@ -84,8 +91,10 @@
 display = Display()
 
 
-def read_key(path):
+def read_key(path, private_key=None):
     try:
+        if private_key:
+            return jwk_from_pem(private_key.encode())
```

Comment:
I would always provide an encoding when using `encode()`:
```suggestion
            return jwk_from_pem(private_key.encode('utf-8'))
```